solution/index.js

```diff
@@ -0,0 +1,209 @@
+let tasks = window.localStorage.getItem("tasks") // all the tasks in the local storage
+let originTask; // saves and pass the <li> element that dragged in to other section  
+let altpressed = false; // global variable to check is alt is currently pressed
+let currentEl; // global variable that contains the <li> tag of the element that the mouse is currently over
```

Comment:
You don't need global variables to know which task is hovered. Find a better way.
Also, there is an easier way to know if alt is pressed during a keyboard event.